static/js/components/JobDetails.jsx

```diff
@@ -65,16 +66,21 @@ var AddForm = React.createClass({
       return (<div><Button bsStyle="primary" onClick={this.addJob}>Add job</Button><p></p></div>);
     }
 
-    var crontime_placeholder = 'Cron time: (e.g.  0 13 */4 * * 1-5 -> run at 0 seconds, 13 mins, every hour divisable by 4, every day and every month on Monday though Friday)';
-    var workload_placeholder = 'Workload: (e.g. --task=job or {"task":"job"})';
+    var crontime_placeholder = 'e.g.  0 13 */4 * * 1-5 -> run at 0 seconds, 13 mins, every hour divisable by 4, every day and every month on Monday though Friday';
+    var workload_placeholder = 'e.g. --task=job or {"task":"job"}';
+    var backend_placeholder = 'e.g. gearman or workflow-mananger';
     var button_msg = this.state.json_workload_checked ? "Yes, submit":"Submit job";
     return (
       <div>
         {this.cronAlert()}
         {this.workloadAlert()}
         <form onSubmit={this.formSubmit} method="POST">
+        <label>Cron Time</label>
         <Input ref="crontime" type="text" placeholder={crontime_placeholder} required />
-        <Input ref="workload" type="text" placeholder={workload_placeholder}/>
+        <label>Workload</label>
+        <Input ref="workload" type="text" placeholder={workload_placeholder} />
+        <label>Backend</label>
+        <Input ref="backend" type="text" placeholder={backend_placeholder} />
```

Comment:
I think we could just make this a `<select>` and pre-fill the two options to avoid confusion. Default to `gearman` for now. It'd be nice not to mess up existing process that people have -> they don't think of gearman as a backend option.